controllers/reviews.controllers.js

```diff
@@ -0,0 +1,25 @@
+const {
+  fetchReviewById,
+  fetchAllReviews,
+} = require("../models/reviews.models");
+
+exports.getAllReviews = (req, res, next) => {
+  fetchAllReviews()
+    .then((reviews) => {
+      res.status(200).send(reviews);
```

Comment:
These need to be sent back as an object, with a key then the data i.e.

`res.status(200).send({reviews:reviews})`